excel_graph_parser/graph_parser.py

```diff
@@ -17,6 +18,9 @@
 
 class ExcelImageParser:
     def __init__(self, excel_file_path: Union[Path, str], params: Munch, from_app: bool = False):
+        warnings.warn("ExcelImageParser is deprecated and will be removed in the future. "
+                      "Please make use of SpreadsheetParser instead.", DeprecationWarning)

```

Comment:
Don't forget to update `SpreadsheetParser` -> `ExcelChartParser`

Reply:
Sharp eye, updated